### DISCUSSION TITLE
Most Used Language's Issue

### DISCUSSION THREAD

**Discussion by shariful10** [anuraghazra/github-readme-stats#3541] [Q&A] [answered]
### Describe the bug

My GitHub Most Used Language's Card Not Updated

### Screenshots / Live demo link

![image](https://github.com/anuraghazra/github-readme-stats/assets/98546214/0ad80f61-bb61-4db9-92b9-2543e691789a)

**Comment by shariful10** [comment]
Python and PHP Not Showing

**Comment by SashaWouse** [comment]
I have the same issue. The statistics widget is also not updating. I tried to update the code, but it didn’t help.

**Comment by qwerty541** [comment, marked as answer]
Hey, @shariful10! Currently we fetch languages data only about 100 oldest repositories. It was implemented this way because iteration through repositories pages on GitHub GraphQL API requires additional consumption of points. Since we're limited to 5000 points per hour it will cause more frequent downtimes of public instance. In future we're going to implement ability to fetch more than 100 repositories under environment variable to allow users enable it on own instances with own API tokens.
